pkg/protocol/handshake/handshake.go

```diff
@@ -61,6 +62,27 @@ type Message interface {
 	Type() Type
 }
 
+// CipherSuiteAuthenticationType controls what authentication method is using during the handshake for a CipherSuite
```

Comment:
@jkralik Do we need to export these? 

It isn't great, but `CipherSuiteKeyExchangeAlgorithm` already exists in `github.com/pion/dtls` so seems confusing to export twice.